TADashboard/src/main/java/page_objects/DataProfilesDisplayFieldsPage.java

```diff
@@ -0,0 +1,23 @@
+package page_objects;
+
+import com.logigear.control.common.imp.CheckBox;
+import com.logigear.control.common.imp.Label;
+import utils.common.Constants;
+
+import java.util.List;
+
+public class DataProfilesDisplayFieldsPage extends BasePage {
+    private Label profileHeader = new Label("css=#profilesettings > tbody > tr > td.profilesettingheader");
+    private Label displayFieldLabels = new Label("css=#profilesettings > tbody label");
+    private CheckBox dynamicCheckBox = new CheckBox("xpath=//table[@id='profilesettings']/tbody//label[contains(text(),'%s')]/input");
+
+    public List<String> getAllDisplayFieldsLabels() {
+        displayFieldLabels.waitForDisplay();
```

Comment:
displayFields

Reply:
Yeah, thank you. I will fix it now

Reply:
I have updated for it. Please check again.

![image](https://user-images.githubusercontent.com/56109274/134302537-ad901179-f338-49d2-9cfd-ff467fc9264b.png)

Reply:
I mean you are missing 's' in Field word of control

Reply:
I have updated it. Please check again. Thank you ^^